pkg/manager/manager.go

```diff
@@ -201,6 +202,10 @@ type Options struct {
 	// LeaseDuration time first.
 	LeaderElectionReleaseOnCancel bool
 
+	// OnStoppedLeading is callled when the leader election lease is lost.
+	// It can be overridden for tests.
+	OnStoppedLeading func()
```

Comment:
You can just run this after the manager ended to achieve the same result or not?

Reply:
lost leader is not equal manager ended. so we can't do that in operator's preStop or other hook.

Reply:
What is the difference? The manager ends after it lost the leader?